instance/factories.py

```diff
@@ -129,6 +129,8 @@ def production_instance_factory(**kwargs):
         # Don't create default users on production instances
         "DEMO_CREATE_STAFF_USER": False,
         "demo_test_users": [],
+        # Disable certificates process to reduce load on RabbitMQ and MySQL
+        "SANDBOX_ENABLE_CERTIFICATES": False,
```

Comment:
We should also include this for sandboxes.  Sandboxes almost never need the certs service, and if we do we can still override the setting via the PR description or via `configuration_extra_settings`.

Reply:
Question: where would one make this a default for sandboxes, if they wanted to? I'm looking around but it's not obvious to me.